src/app/utilities/auth/constants.ts

```diff
@@ -48,3 +48,28 @@ export const DEFAULT_USER = {
 export const USER_TABLE_NAME = process.env.USER_TABLE_NAME || 'NDAUsers';
 export const USER_TABLE_PARTITION_ID = 'Email';
 export const DEFAULT_SIGN_IN_REDIRECT_URL = '/profile';
+
+export const OPTIONS_DATE = Array.from({ length: 31 }, (_, i) => ({
+  label: (i + 1).toString(),
+  value: i + 1,
+}));
+
+export const OPTIONS_MONTH = [
+  { label: 'Jan', value: '1' },
```

Comment:
Use `number` type for `OPTIONS_MONTH`'s `value`s. `Date()` constructor probably expects `monthIndex`, so the value should start from '0`.